# UTM Information about the working area
utm_zone: "18"
utm_band: "T"

# A Point in the AB Line
# This is also the assumed starting
# of the tractor
a:
  latitude: 45.50583650   # deg
  longitude: -73.57913300 # deg
  altitude: 0             # m

# B Point in the AB Line
b:
  latitude: 45.50586400   # deg
  longitude: -73.57916650 # deg
  altitude: 0             # m

# This is the point where the tractor
# will enter the field
entry_position:
  latitude: 45.50583650   # deg
  longitude: -73.57913300 # deg
  altitude: 0             # m

# The buffer distance required between
# the generated paths and the boundary
headland_width: 0.10       # m

# The width of the tool's effective area
tool_width: 0.2           # m

# The boundary of the area to be covered
# This is an ORDERED (CCW) list of points
# that creates a polygon used as a bound
# All points should follow this format:
#   latitude: _____ [deg]
#   longitude: ____ [deg]
#   altitude: _____ [m]
boundary:
  points:
    - latitude: 45.50583650
      longitude: -73.57913300
      altitude: 0
    - latitude: 45.50584717
      longitude: -73.57911617
      altitude: 0
    - latitude: 45.50587583
      longitude: -73.57914733
      altitude: 0
    - latitude: 45.50586400
      longitude: -73.57916650
      altitude: 0
